docker-compose.yml

```diff
@@ -20,11 +20,11 @@
 
 web:
   build: .
-  command: python run.py --initdb
+  command: bash -c "claimstore initdb && claimstore populatedb && python run.py"
```

Comment:
This could be moved to a dedicated `scripts` directory from which both python venv and docker could run the stuff out.  This can be refactored later.